### PR TITLE
add known issues

### PR DESCRIPTION
**Pull request by JacobGros**
# Description
A few sentences describing the overall goals of the pull request's commits.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/714 |
https://github.com/dell/csm/issues/713

# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have maintained backward compatibility

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.6.0.md [hunk 1]
```diff
@@ -60,3 +60,8 @@
 - vSphere : Can not create VM host object. ([#592](https://github.com/dell/csm/issues/592))
 - PowerScale Replication: Artifacts are not properly cleaned after deletion. ([#523](https://github.com/dell/csm/issues/523))
 - PVC fails to resize with the message spec.capacity[storage]: Invalid value: "0": must be greater than zero. ([#507](https://github.com/dell/csm/issues/507))
+
+#### Known Issues
+
+- PowerFlex driver deployment via CSM Operator should work without having the user to create ConfigMap([#713](https://github.com/dell/csm/issues/713))
+- Powerflex driver specific parameters mentioned in the samples are not configurable when deployed via CSM Operator ([#714](https://github.com/dell/csm/issues/714))
```
Comment (bharathsreekanth) [inline CHANGELOG/CHANGELOG-1.6.0.md:67]: Could I ask a favor: Could you also add the below line since it is not captured here

Comment (bharathsreekanth) [reply CHANGELOG/CHANGELOG-1.6.0.md:67]: CSM object does not track available deployment count when down scaling to n-1 , where n is number of nodes.

**Review by bharathsreekanth — APPROVED** [review]

**Comment by bharathsreekanth** [comment]
@gallacher Could you review this PR and make sure we have what is needed? Thanks

**Comment by shaynafinocchiaro** [comment]
Why are these github issues being added manually?

**Comment by bharathsreekanth** [comment]
> Why are these github issues being added manually?

Sean G asked us to do it this way given they came about today and probably did not want to wait until the BOT runs again.

**Comment by shaynafinocchiaro** [comment]
If the github issues have the appropriate labels/milestone, we can run the automation out of cycle to get these in the changelog. The changelog-staging branch is deleted after merge so may not want to merge into that branch.

**Comment by JacobGros** [comment]
Can we close this PR? 
@bharathsreekanth @shaynafinocchiaro

**Comment by sharmilarama** [comment]
@JacobGros yes, we can rerun the automation.

**Comment by JacobGros** [comment]
Closing PR for automation

**Pull request closed without merging**